torch/_inductor/codegen/common.py

```diff
@@ -349,6 +349,10 @@ def _print_ModularIndexing(self, expr):
 
     def _print_FloorDiv(self, expr):
         x, div = expr.args
+        if div == -1:
+            return f"{self.paren(self.doprint(x * -1))}"
```

Comment:
Can we handle this in `FloorDiv.eval`? That would potentially allow SymPy to make further simplifications.

Reply:
Moved.